src/arch/x86_64/CodeGen.zig

```diff
@@ -1318,7 +1318,7 @@ fn airAddWrap(self: *Self, inst: Air.Inst.Index) !void {
     const result: MCValue = if (self.liveness.isUnused(inst))
         .dead
     else
-        return self.fail("TODO implement addwrap for {}", .{self.target.cpu.arch});
+        try self.genBinMathOp(inst, bin_op.lhs, bin_op.rhs);
```

Comment:
this function now is identical to `airAdd`, correct? I suggest to delete one of them and have the big switch map both `add` and `add_wrap` to the same function.

Reply:
Done!